modules/passkey/test/libraries/WebAuthn.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { expect } from 'chai'
+import { deployments, ethers } from 'hardhat'
+import hre from 'hardhat'
+import { DUMMY_CLIENT_DATA_FIELDS, DUMMY_SIGNATURE_BYTES, base64UrlEncode, getSignatureBytes } from '../utils/webauthn'
+
+describe('WebAuthn Library', () => {
+  const setupTests = deployments.createFixture(async ({ deployments }) => {
+    const { FCLP256Verifier } = await deployments.fixture()
+    const verifier = await ethers.getContractAt('FCLP256Verifier', FCLP256Verifier.address)
+
+    const WebAuthnLibFactory = await ethers.getContractFactory('TestWebAuthnLib')
+    const deployedWebAuthnLib = await WebAuthnLibFactory.deploy()
+    const webAuthnLib = await ethers.getContractAt('TestWebAuthnLib', await deployedWebAuthnLib.getAddress())
+    const mockP256Verifier = await (await hre.ethers.getContractFactory('MockContract')).deploy()
+
+    return { webAuthnLib, verifier, mockP256Verifier }
+  })
+
+  it('Should return false when the verifier returns false', async () => {
+    const { webAuthnLib, verifier } = await setupTests()
```

Comment:
Nit: This is the only test using an actual verifier, and relies on the signature being invalid. Why not just use the Mock Verifier here again?

Reply:
done